src/engine/types/actions.rs

```diff
@@ -685,3 +685,38 @@ impl<T> EngineAction for StopAction<T> {
         }
     }
 }
+
+/// Action indicating the result of growing a block device or block devices in a pool.
+pub enum GrowAction<T> {
+    Identity,
+    Grown(T),
+}
+
+impl Display for GrowAction<DevUuid> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            GrowAction::Identity => write!(
+                f,
+                "No changes to block device size were detected; no action taken"
+            ),
+            GrowAction::Grown(uuid) => {
+                write!(f, "Block device with UUID {} was successfully grown", uuid,)
```

Comment:
We agreed in meeting that adding the pool UUID to this log message might help out a user who needed to related a previous CLI command to log messages.

Reply:
This should be resolved now!